chapter02/qli/ex5.pl

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/perl
+
+use warnings;
+
+my $str=<STDIN>;
+chomp($str);
+
+my $num=<STDIN>;
+chomp($num);
+
+print "Ex Result:: \n";
+if ( $num && $num>0 ){
```

Comment:
Yes, thanks, I could remove both the chomp on the $str and the newline. As new line character will be picked by the input anyway.